atds.opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+maintainer: "martin@mjambon.com"
+authors: ["Martin Jambon"]
```

Comment:
You should put your name here.

Reply:
Do you mean in the authors, or also maintainer?

I've actually never used opam before... The last time I was working mostly in OCaml, was back when oasis was just starting to catch on. 

I was assuming you would publish to opam along with the other packages when doing a release, but if you'd prefer I publish atds, I can look into how that works.

Reply:
I mean the authors.

Other people can take care of the rote aspects of publishing releases. What we'll need from you is that you update the root `CHANGES.md` with changes occurring in atds.